### PR TITLE
Add methods for transformation jobs cancelation

### PR DESCRIPTION
**Pull request by silvavelosa**
## Description
add transformationAPI.cancel() method

## Checklist:
- [ ✔️ ] Tests added/updated.
- [ ✔️ ] Documentation updated. Documentation is generated from docstrings - these must be updated according to your change. 
  If a new method has been added it should be referenced in [cognite.rst](https://github.com/cognitedata/cognite-sdk-python/blob/master/docs/source/cognite.rst) in order to generate docs based on its docstring. 
- [ ✔️ ] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/cognite-sdk-python/blob/master/CHANGELOG.md).
- [ ✔️ ] Version bumped. If triggering a new release is desired, bump the version number in [_version.py](https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_version.py) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: cognite/client/data_classes/transformations/jobs.py [hunk 1]
```diff
@@ -109,6 +109,12 @@ def update(self):
         self.finished_time = updated.finished_time
         self.last_seen_time = updated.last_seen_time
 
+    def cancel(self):
+        if self.transformation_id is None:
+            self._cognite_client.transformations.cancel(transformation_external_id=self.transformation_external_id)
+        else:
+            self._cognite_client.transformations.cancel(transformation_id=self.transformation_id)
+
```
Comment (emelvarol-cognite) [inline cognite/client/data_classes/transformations/jobs.py:117]: nice to also support in jobs!

**Review by emelvarol-cognite — APPROVED** [review]
great!

**Comment by emelvarol-cognite** [comment]
my review is not enough but lgtm :D

**Review by erlendvollset — APPROVED** [review]
LGTM

**Pull request merged**